deployments/setup-fork.ts

```diff
@@ -0,0 +1,210 @@
+    console.log(`Funding ${account}...`);
+
+    for (const fundingRequest of fundingRequests) {
+        if (fundingRequest.token === NATIVE_TOKEN_ADDRESS) {
+            await fundingRequest.whale.sendTransaction({
+                value: fundingRequest.amount,
+                to: account
+            });
+
+            continue;
+        }
+
+        const tokenContract = await Contracts.ERC20.attach(fundingRequest.token);
+        await tokenContract.connect(fundingRequest.whale).transfer(account, fundingRequest.amount);
+    }
+};
+
+const removeDepositLimits = async (tokens: string[]) => {
+    console.log('Removing deposit limits...');
+    console.log();
+
+    const { deployer } = await getNamedSigners();
+
+    const poolCollection = await DeployedContracts.PoolCollectionType1V1.deployed();
+    for (const token of tokens) {
+        await poolCollection.connect(deployer).setDepositLimit(token, MAX_UINT256);
+    }
+};
+
+const setLockDuration = async (lockDuration: number) => {
+    console.log(`Setting withdrawal lock duration to ${lockDuration} seconds...`);
+    console.log();
+
+    const { deployer } = await getNamedSigners();
+
+    const pendingWithdrawals = await DeployedContracts.PendingWithdrawals.deployed();
+    await pendingWithdrawals.connect(deployer).setLockDuration(lockDuration);
+};
+
+const archiveArtifacts = async () => {
+    const zip = new AdmZip();
+
+    const srcDir = path.resolve(path.join(__dirname, './tenderly'));
+    const dest = path.resolve(path.join(__dirname, `../fork-${new Date().toISOString()}.zip`));
+
+    zip.addLocalFolder(srcDir);
+    zip.writeZip(dest);
+
+    console.log(`Archived ${srcDir} to ${dest}...`);
+    console.log();
+};
+
+const main = async () => {
+    if (!isTenderlyFork()) {
+        throw new Error('Invalid network');
+    }
+
+    console.log();
+
+    const forkId = await createTenderlyFork();
+
+    const { dai, link } = await getNamedAccounts();
+    const { ethWhale, bntWhale, daiWhale, linkWhale } = await getNamedSigners();
+    const bnt = await DeployedContracts.BNT.deployed();
+
+    const ethAmount = 10_000;
+    const bntAmount = 10_000;
+    const daiAmount = 500_000;
+    const linkAmount = 10_000;
+
+    const fundingRequests = [
+        {
+            token: NATIVE_TOKEN_ADDRESS,
+            amount: toWei(ethAmount),
+            whale: ethWhale
+        },
+        {
+            token: bnt.address,
+            amount: toWei(bntAmount),
+            whale: bntWhale
+        },
+        {
+            token: dai,
+            amount: toWei(daiAmount),
+            whale: daiWhale
+        },
+        {
+            token: link,
+            amount: toWei(linkAmount),
+            whale: linkWhale
+        }
+    ];
+
+    const devAddresses = DEV_ADDRESSES.split(',');
+
+    for (const account of devAddresses) {
+        await fundAccount(account, fundingRequests);
+    }
+
+    console.log();
+
+    const lockDuration = 2;
```

Comment:
I'd put this in the `EnvOptions` object, right next to `FORK_RESEARCH: isResearch`, since they are kind of coupled together anyway.

Reply:
It's business logic - not an external configuration